models/examples/secondary-care/community/acute/nominated-pharmacy/clinical-practitioner/1-Task-Request-Return-200_OK.json

```diff
@@ -35,19 +26,19 @@
       "value": "9446368138"
     }
   },
-  "authoredOn": "2020-12-21T17:03:20-00:00",
+  "authoredOn": "2016-03-10T22:39:32-04:00",
   "owner": {
     "identifier": {
-      "system": "https://fhir.hl7.org.uk/Id/gphc-number",
-      "value": "7654321"
+      "system": "https://fhir.nhs.uk/Id/ods-organization-code",
+      "value": "VNE51"
```

Comment:
Community pharmacy code rather than HAH here, we've used FH542 for other examples

Reply:
This is just Kevin's example from the IG. Happy to swap it out though

Reply:
Yeah, makes sense to match the nominated pharmacy of the original prescription

Reply:
Updated ODS code, NHS number and short form prescription ID. We'll need to update the referenced message IDs at some point, but we don't have examples for these to match yet

Reply:
Good shout, will check dispense notification ids line up too